src/api/liskApi.js

```diff
@@ -12,183 +12,177 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-
-/**
- * LiskAPI module provides functions for interfacing with the Lisk network.
- * Providing mechanisms for:
- *
- * - Retrieval of blockchain data: accounts, blocks, transactions.
- * - Enhancing Lisk security by local signing of transactions and immediate network transmission.
- * - Connecting to Lisk nodes or to localhost instance of Lisk core.
- * - Configurable network settings to work in different Lisk environments.
- *
- *     var lisk = require('lisk-js');
- *
- *     var options = {
- *         ssl: true,
- *         node: '',
- *         randomNode: true,
- *         testnet: true,
- *         port: '7000',
- *         bannedNodes: [],
- *         nodes: [],
- *         nethash: ''
- *     };
- *
- *
- *     var LSK = lisk.api(options);
-*/
 import config from '../../config.json';
-import { LIVE_PORT, TEST_PORT, GET, POST } from '../constants';
+import { LIVE_PORT, SSL_PORT, TEST_PORT, GET, POST } from '../constants';
 import * as privateApi from './privateApi';
 import * as utils from './utils';
 
-/**
-*
-* @class LiskAPI
-* @param {Object} options - Initialization Object for the LiskAPI instance.
- */
+const bannedNodes = Symbol('bannedNodes');
+const defaultNodes = Symbol('defaultNodes');
+const defaultSSLNodes = Symbol('defaultSSLNodes');
+const defaultTestnetNodes = Symbol('defaultTestnetNodes');
+const nethash = Symbol('nethash');
+const ssl = Symbol('ssl');
+const testnet = Symbol('testnet');
```

Comment:
what is the reason to set Symbol for these?
It might be solved by introducing flow?
https://github.com/LiskHQ/lisk-js/pull/556/files#diff-3aefae204ccf5f3f6e65ce8890b8c25cR45

Reply:
It's a way of making them private since there's no way to access the symbol outside of the module.